data/json/monsters/zed-classic.json

```diff
@@ -23,6 +23,7 @@
     "armor_bash": 6,
     "armor_cut": 8,
     "armor_bullet": 6,
+    "weakpoints": [ { "name": "the eye", "armor_mult": { "physical": 0 }, "coverage": 1 } ],
```

Comment:
I feel this weak point should be renamed to something like "viewing slit" or "eye slit" or similar. Otherwise it might confuse players who will think that eye is the zombie's weak point, and AFAIK according to our lore zombies don't have any weak points at all (headless zombie will confirm).

Reply:
Calling it a "viewing slit" is pretty stilted and inaccurate on most of these creatures. The eye is a weakpoint not because they feel extra pain, but because it's not likely to have armour protecting it in most cases. For example, a scarred zombie doesn't have scar tissue covering its eye.

I've generally avoided listing organs. The body parts included here are separate because hitting them might confer another effect, like staggering the zombie, once we have more effects. It should be noted though that our zombies don't have "vulnerable spots" in the sense that destroying the head kills them for good, but they do canonically have functional organs that can be temporarily taken out of commission to the zombie's disadvantage. That's why bleeding works on most of them for example.